### ISSUE TITLE
Undefined Namespace

### ISSUE THREAD

**Issue by MHCreations** [kalessil/phpinspectionsea#1490]
| Subject       Undefined Namespace 
| Plugin         | Php Inspections (EA Extended) 4.0.3 |
| Language level |PHP 7.2                   |

Current behaviour
--
Example:

header.php is an include file that states: 

`Use path\to\classes\Bash`

And the main PHP page has: 

```
include header.php;

$newObject = Bash\ClassName();
```

But EA Inspector states: 

> "Undefined namespace 'Bash' <br>
 Inspection info: Reports the references to a namespace that is not found." 

Expected behaviour
--
The Class should be findable. 

This WORKS:
```
 include header.php;
Use path\to\classes\Bash
$newObject = Bash\ClassName();
```

So ultimately, can EA Inspector search includes or include paths or resource paths for unfound Namespaces?

Many thanks!

Environment details:
PHPStorm 2019.3.1.
$_SERVER['DOCUMENT_ROOT'] has been correctly set. 
include paths in PhpStorm settings have been correctly set.
Resource paths in PhpStorm have been correctly set. 

---

**Comment by MHCreations** [comment]
The use keyword must be declared in the outermost scope of a file (the global scope) or inside namespace declarations. This is because the importing is done at compile time and not runtime, so it cannot be block scoped. The following example will show an illegal use of the use keyword: 

Sorry,  I made the mistake. Please remove this report. Thanks

**Comment by kalessil** [comment]
No worries @MHCreations, closing the issue.
